### PR TITLE
Sus020 rh headers and footers

### PR DESCRIPTION
**Pull request by jrbarnes9**
# Motivation and Context
Update headers and footers on RH to use those in [sdc-pattern library](https://sdc-global-design-patterns.netlify.com/components/detail/header.html) to ensure user experience is consistent when passing to eQ.

# What has changed
- Replaced page header with pattern library component
- Replaced page footer with pattern library component 

Additional pattern library markup alignment:
- Updated head to use styles and images from latest version of patter library on cdn
- Replaced js scripts in RH to use those from pattern library on CDN
- Updated markup for confidentiality message to use pattern library markup
- Added PL 'skip to content' accessibility link

Bug fixes:
- Fixed broken error message anchor link

Content changes:
- Updated page content to improve UX/readability
- Pointed 'what we do' footer link to household studies 'about' content

# How to test?
- Check header and footers are correct and links work
- Cross browser test and code validation
- Check content is as agreed with social team
- Check required javascript functions still work

# Links
https://trello.com/c/UwNvY2vg/283-sus020-rh-headers-and-footers-5

# Screenshots (if appropriate):

### CODE REVIEW

File: app/templates/partials/footer.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<footer class="footer" role="contentinfo">
+    <div class="container">
+        <div class="grid">
+            <div class="grid__col col-4@m">
+                <p class="venus footer__heading"><strong>Legal information</strong></p>
+                <ul class="list list--bare">
+                    <li>
+                        <a href="/cookies-privacy"
```
Comment (jcox-dev) [inline app/templates/partials/footer.html]: Should this be pointing at: "https://surveys.ons.gov.uk/cookies-privacy"? As that route doesn't exist in RH.

File: app/templates/partials/footer.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+            <div class="grid__col col-4@m">
+                <p class="venus footer__heading"><strong>About ONS</strong></p>
+                <ul class="list list--bare">
+                    <li>
+                        <a href="https://www.ons.gov.uk/surveys/informationforhouseholdsandindividuals/householdandindividualsurveys"
+                           class="footer__link">What we do</a>
+                    </li>
+                    <li>
+                        <a href="/contact-us"
```
Comment (jcox-dev) [inline app/templates/partials/footer.html]: Likewise, there is no `/contact-us` path. "https://www.ons.gov.uk/aboutus/contactus"?

File: app/templates/partials/footer.html (hunk 1 above)
Comment (jrbarnes9) [reply app/templates/partials/footer.html]: @jcox-dev I think the plan is to duplicate the privacy page so RH has it's own social privacy page has it's own  (due to issues with business 'sign in' link etc on https://surveys.ons.gov.uk/cookies-privacy).  (https://trello.com/c/d0U2sz8L/278-sus021-update-to-content-of-surveys-privacy-page-to-include-information-for-social-surveys)

**Comment by jcox-dev** [comment]
[SUS021 Update to content of Surveys Privacy page to include information for Social Surveys](https://trello.com/c/d0U2sz8L/278-sus021-update-to-content-of-surveys-privacy-page-to-include-information-for-social-surveys)

File: app/templates/partials/footer.html (hunk 1 above)
Comment (jrbarnes9) [reply app/templates/partials/footer.html]: @jcox-dev I'll revert these links and leave that work for sus021.

**Pull request merged**